GameDev/week7/livestream/script.md

```diff
@@ -1 +1,109 @@
-Put your script here.
+Particle System basics (10 mins)
```

Comment:
Awesome content, but can you make it more readable? I know it's a script though, so that may be more challenging. Maybe split it into sections?